forest/src/daemon.rs

```diff
@@ -72,9 +73,14 @@ pub(super) async fn start(config: Config) {
     let coeff = config.drand_public;
 
     // TODO: Interval is supposed to be consistent with fils epoch interval length, but not yet defined
-    let beacon = DrandBeacon::new(coeff, genesis.blocks()[0].timestamp(), 1)
-        .await
-        .unwrap();
+    let beacon = DrandBeacon::new(
+        "https://api.drand.sh",
```

Comment:
This is used in 2 places might be better to make it a constant

Reply:
yeah I'll do that, was just being lazy since the one is just used for tests that are ignored by default